packages/loaders/src/elements/Dots.tsx

```diff
@@ -39,6 +39,9 @@ export interface IDotsProps extends React.HTMLAttributes<SVGSVGElement> {
   delayMS?: number;
 }
 
+/**
+ * @extends HTMLAttributes<SVGSVGElement>
```

Comment:
I think `SVGAttributes<SVGSVGElement>` is preferable, and the `interface` should be modified accordingly.